components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2176,4 +2176,49 @@ void RewardsServiceImpl::SetShortRetries(bool short_retries) {
   bat_ledger_service_->SetShortRetries(short_retries);
 }
 
+bool SavePendingContributionOnFileTaskRunner(PublisherInfoDatabase* backend,
+    const ledger::PendingContributionList& list) {
+  if (!backend) {
+    return 0;
```

Comment:
returning 0 is fine; but I think you meant to return `false` (since it's a `bool`)

Reply:
fixed